File: Microservices/IoT.House.Automation.Microservices.Arduino/IoT.House.Automation.Microservices.Arduino.Application/MessageBroker/Events/ArduinoRemovedEvent.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IoT.House.Automation.Microservices.Arduino.Application.MessageBroker.Events
{
    public class ArduinoRemovedEvent : Event
    {
        public Guid ArduinoIdentifier { get; set; }
        public override string Type => this.GetType().Name;

        public ArduinoRemovedEvent(Guid arduinoIdentifier)
        {
            ArduinoIdentifier = arduinoIdentifier;
        }
    }
}

File: Microservices/IoT.House.Automation.Microservices.Mobile/application/src/main/java/iot/house/automation/microservices/mobile/application/messaging/events/ArduinoRemovedEvent.java
package iot.house.automation.microservices.mobile.application.messaging.events;

import lombok.Getter;

import java.util.UUID;

@Getter
public class ArduinoRemovedEvent extends BaseEvent {
    private UUID arduinoIdentifier;

    public ArduinoRemovedEvent(UUID arduinoIdentifier) {
        this.arduinoIdentifier = arduinoIdentifier;
    }

    @Override
    public String getType() {
        return this.getClass().getSimpleName();
    }
}
